components/TransactionsPage/transaction-card.js

```diff
@@ -0,0 +1,34 @@
+import React,{useState, useEffect} from 'react';
+import { TransactionInfo } from './transactions-info';
+import axios from 'axios';
+
+export function TransactionCard({data={}}) {
+  const [isOpen, setisOpen] = useState(false);
+
+  function closeModal() {
+    setisOpen(false);
+  }
+    const formatter = (date) =>{
+      return date.slice(11,16)
+   }
```

Comment:
postDate value is like this -> "2022-09-06T00:00:00Z" Why do you get slice(11, 16), you should slice(0, 10)

![image](https://user-images.githubusercontent.com/97659632/188594685-19f00c20-f416-42a6-b525-a4ba912cfe28.png)